### PR TITLE
[DOCS] Fix links to .NET and PHP clients

### PR DESCRIPTION
**Pull request by lcawl**
Relates to https://github.com/elastic/docs/issues/2518

This PR replaces hard-coded links with appropriate shared attributes.

### CODE REVIEW

**Comment by lcawl** [comment]
@elasticmachine update branch

**Review by kilfoyle — APPROVED** [review]
LGTM! ⛵

File: x-pack/docs/en/security/ccs-clients-integrations/http.asciidoc [hunk 1]
```diff
@@ -85,8 +85,8 @@ specific clients, refer to:
 
 * {java-api-client}/_basic_authentication.html[Java]
 * {jsclient-current}/auth-reference.html[JavaScript]
-* https://www.elastic.co/guide/en/elasticsearch/client/net-api/master/configuration-options.html[.NET]
+* {es-dotnet-client}/configuration-options.html[.NET]
```
Comment (lcawl) [inline x-pack/docs/en/security/ccs-clients-integrations/http.asciidoc:88]:
```suggestion
* {es-dotnet-client}/configuration.html[.NET]
```

File: x-pack/docs/en/security/ccs-clients-integrations/http.asciidoc [hunk 2]
```diff
@@ -85,8 +85,8 @@ specific clients, refer to:
...
 * https://metacpan.org/pod/Search::Elasticsearch::Cxn::HTTPTiny#CONFIGURATION[Perl]
-* https://www.elastic.co/guide/en/elasticsearch/client/php-api/master/security.html[PHP]
+* {es-php-client}/security.html[PHP]
```
Comment (lcawl) [inline x-pack/docs/en/security/ccs-clients-integrations/http.asciidoc:90]:
```suggestion
* {es-php-client}/connecting.html[PHP]
```

**Comment by lcawl** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.5|[](https://github.com/elastic/elasticsearch/pull/90432)|
|✅|8.4|[](https://github.com/elastic/elasticsearch/pull/90433)|
|✅|8.3|[](https://github.com/elastic/elasticsearch/pull/90434)|
|✅|8.2|[](https://github.com/elastic/elasticsearch/pull/90435)|
|✅|8.1|[](https://github.com/elastic/elasticsearch/pull/90436)|
|✅|8.0|[](https://github.com/elastic/elasticsearch/pull/90437)|
|✅|7.17|[](https://github.com/elastic/elasticsearch/pull/90438)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**